Library/Formula/xmlto.rb

```diff
@@ -15,6 +15,8 @@ class Xmlto < Formula
   depends_on 'docbook'
   depends_on 'docbook-xsl'
   depends_on 'gnu-getopt'
+  # Doesn't strictly depend on GNU getopt, but OS X system getopt(1)
+  # does not suport longopts in the optstring, so use GNU getopt.
 
```

Comment:
See [line 23 of xmlto](https://fedorahosted.org/xmlto/browser/xmlto.in#L23); there are other implementations that support long options. See, for example, [ksh getopts](http://www2.research.att.com/sw/download/man/man1/getopts.html), and [getopt_long(3)](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man3/getopt_long.3.html), which could readily be turned into a shell utility with a simple main function. I thought this comment would be helpful, so people know this isn't strictly speaking a dependency on a specific package, just a specific _feature_.